uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/app-dashboard.js

```diff
@@ -52,9 +52,7 @@ define([
                       templateUrl: config.path.TPL + 'dashboard/documents.html?v=8',
                       resolve: {
                         documentList: ['DocumentService', function (DocumentService) {
-                          return DocumentService.get({
-                            'status_id': { 'NOT IN': config.status.resolve.DOCUMENT }
-                          });
+                          return DocumentService.get({});
```

Comment:
@brunoqueiros Did you verify with Mike, that why previously "rejected" and "approved" documents were not fetched?